src/sentry/integrations/gitlab/integration.py

```diff
@@ -298,22 +319,23 @@ def build_integration(self, state):
         secret = sha1_text("".join([hostname, state["installation_data"]["client_id"]]))
 
         integration = {
-            "name": group["full_name"],
+            "name": group.get("full_name", hostname),
             # Splice the gitlab host and project together to
             # act as unique link between a gitlab instance, group + sentry.
             # This value is embedded then in the webook token that we
             # give to gitlab to allow us to find the integration a hook came
             # from.
-            "external_id": u"{}:{}".format(hostname, group["id"]),
+            "external_id": u"{}:{}".format(hostname, group.get("id", "instance")),
```

Comment:
This could conflict with a group named `instance`. We could use `_instance_` to make that clash less likely.

Reply:
Good point. I've changed the default value to `_instance_`.

Reply:
Hmmm, did we accidentally go from `_instance_` -> `instance` during the rebase? Can we change it back? Or was it intentional?

Reply:
@kinghuang Do you have an answer to Manu's question?